src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+    
+    private boolean isAdded; //If this value is false, it means the tag has been deleted.
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.